packages/generator/src/typedoc.ts

```diff
@@ -73,7 +73,7 @@ export function getNavPropertyDescription(
   property: VdmNavigationProperty
 ): string {
   return `${
-    property.isMultiLink ? 'One-to-many' : 'One-to-one'
+    property.isCollection ? 'One-to-many' : 'One-to-one'
```

Comment:
I know it si only doc, but perhaps align it with the `1 -1` and `1-N` case mentioned above.

Reply:
I don't get this comment? Can you explain a little more what you mean?

Reply:
The other multiplicity is deprecated so ignore the comment.